config/constants.ts

```diff
@@ -63,8 +63,8 @@ export const eventBusName = 'OrcaBusMain';
 export const eventSourceQueueName = 'orcabus-event-source-queue';
 
 // Event Archiver constants for EventBus Contruct in SharedStack
-export const archiveBucketName = 'event-archive-bucket';
```

Comment:
Perhaps we should put the app namespace (`orcabus`) here as well, so #233 does not happen again?

Reply:
added, have a look now

Reply:
@reisingerf must review bucket naming

Reply:
We usually follow the convention to add an account suffix, e.g. `orcabus-event-archive-dev`
I also think we can remove the `-bucket` part (it's the name for a bucket, so that's implicit).

Reply:
@reisingerf sure, bucket name convention fixed with prefix + account suffix, `orcabus-event-archive-dev`, `orcabus-event-archive-stg`, `orcabus-event-archive-prod`

Reply:
Hold on. Can we have a quick call for this, pls.? 

Is it account suffix or prefix? And, whether to use AccountID or Account Name alias.

Just an example:
```
12345678901-orcabus-event-archive   --> dev
12345678902-orcabus-event-archive   --> stg
12345678903-orcabus-event-archive   --> prod
```

Reply:
sure, we can have a call. 
do we need AccountID as the prefix?  as the env 'dev', 'stg', and 'prod' suffix already show env.

Reply:
bucket name has been converted to `<accountId>-orcabus-event-archive` as we discussed.

Reply:
Hang in there, Ray. 

Please be patient with us. We are improving this bucket naming tinkering as we goes.! Your work here is vital & valuable for this move.

We are getting there... :)

Reply:
Great thanks. Happy to contribute and will update the naming when is ready.